docs/source/tutorials/Install Myria3D on WSL2 with CUDA support.md

```diff
@@ -0,0 +1,132 @@
+    - If the cuda version is 11.3.1 (or the same version stated in the environemnt.yml file), then you can carry on with the installation process without changing anything.
+    - If the cuda version is something else, then you need to cross-check with [this repository](https://data.pyg.org/whl/) if it is supported. At the time of writing this documentation, cuda 9.2, 10.0, 10.1, 10.2, 11.1, 11.3, 11.5, 11.6, 11.7 these versions have corresonding PyG (PyTorch Geometric) libraries to run Myria3D.
+        - If your cuda version is supported, then follow the next steps, otherwise install a driver version that is supported. You can check [here](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#id4) which driver version is required for the cuda version you need, then also check if that is supported by your GPU model.
+        - After you figured out and installed the compatible driver on your windows machine, you need to change the cuda version in the environment.yml file as well.
+        - Comment out the following line, as we are using WSL, we should not install the cuda driver. 
+
+                conda-forge::cudatoolkit=11.3.1
+        - Change the following links depending on which cuda version and pytorch version you are using
+
+                Wheels for torch-geometric optionnal dependencies
+                - https://data.pyg.org/whl/torch-1.11.0+cu113/torch_cluster-1.6.0-cp39-cp39-linux_x86_64.whl
+                - https://data.pyg.org/whl/torch-1.11.0+cu113/torch_scatter-2.0.9-cp39-cp39-linux_x86_64.whl
+                - https://data.pyg.org/whl/torch-1.11.0+cu113/torch_sparse-0.6.14-cp39-cp39-linux_x86_64.whl
+        - Change the following link depending on your cuda version. Check the compatibility tables [here](https://github.com/pyg-team/pytorch_geometric#pip-wheels) and change the git version accordingly. At the time of writing this, PyG (PyTorch Geometric) was in version PyG 2.2.0 and it supports cuda 10.2, 11.3, 11.6 and 11.7 on Linux. If you have any of these cuda, then change the git version to 2.2.0 otherwise keep it at 2.1.0 
+
+                - git+https://github.com/pyg-team/pytorch_geometric.git@2.1.0
+
+3. We use anaconda to manage virtual environments. This makes installing pytorch-related libraries way easier than using pure pip installs.
+
+4. To install the virtual environment, run the following commands within Ubuntu/ WSL terminal-
+
+        conda env create -f "/PATH/TO/environment.yml"
+
+    It will take couple of minutes to download and install all the packages. After that activate the environment by running-
+
+        conda activate myria3d
+5. Then install from a specific branch from github directly.
+
+        pip install --upgrade https://github.com/IGNF/myria3d/tarball/main
+
+At this point Myria3D is installed and you can move ahead with inference or testing using the method stated [here](https://ignf.github.io/myria3d/tutorials/make_predictions.html) (CPU-only).
```

Comment:
Nice! Could you please add that at this points CUDA is not yet installed, but that is is done in the next section? Just to be a bit more explicit :)